Nautilus/Documentation/guides/assetbundles.md

```diff
@@ -0,0 +1,114 @@
+
+Basic modding knowledge is required for this tutorial.
+Here are some things you will need to know:
+- Basic modding and C# knowledge
+- Basic knowledge of how to navigate the Unity game engine interface
+
+The first thing you will need to do (Assuming you have a model already) is download the Unity Hub.  
+This is where you will get the Unity version you need to use to export your asset bundle.
+
+The version Subnautica uses is 2019.4.36 for SN1, as well as Below Zero.
+If you can't find that exact version, just get the one closest to it.
+
+Here are the download links you will need:  
+- Unity hub: [https://unity.com/download](https://unity.com/download)
+- Unity version archive: [https://unity.com/releases/editor/archive](https://unity.com/releases/editor/archive)
+
+Once you've installed both, open Unity hub and click "New Project" from the top right.  
+Once there, select 3D Core, choose your project name and then click "Create Project".  
+![assetbundles-project-setup](../images/guides/assetbundles-project-setup.png)
+
+> [!NOTE]
+> If you get the error "<font color="red">Failed to resolve project template</font>", follow these steps:
+> 1. Locate where your editor is. 
+> You can do that by opening Unity Hub, going to Installs, and then right clicking on the version in question and clicking "Show in Explorer"
+> 2. Go to ``Editor\Data\Resources\PackageManager\ProjectTemplates``. You will find a ``manifest.json`` and ``UnityLicense.json`` files. delete them both.
+> 3. Kill Unity Hub in your task manager and open it again.
+> 
+> You should now be able to create your project!
+
+The Unity editor will now open. This may take a while.  
+Once the editor is open, drag your model from your file manager into the Scene window.
+
+You can either click on your model in the scene window or in the hierarchy on the left to select it.
+
+Next, click on ``Window`` at the top of the Unity editor, and then click ``Package Manager``.  
+Click on the dropdown at the top left and select ``Unity Registry`` if not on it already.  
+Search for "Asset Bundle Browser" and install it.
+
+Next, drag your model from the hierarchy on the left into the ``Assets`` section at the bottom.  
+This will create what is called a ``Prefab``. It can be instantiated (created/spawned) multiple times, but they are all copies of the original.
+
+To edit the prefab, double click on it in the ``Assets`` window.  
+You can add components to the prefab by clicking "Add Component" in the inspector on the right.  
+One example of a component is a ``RigidBody``, which can apply physics to your Prefab.  
+Any components you add will be included in the AssetBundle.
+
+You <font color="red">**should not**</font>, however, add scripts you made in the editor, as the AssetBundle will not link them to the scripts in your modding project and you will get errors.
```

Comment:
Great warning, I remember trying this out myself and being very disappointed.

Want to put it in an "[!IMPORTANT]" note and move it around a bit?